CONTRIBUTING.md

```diff
@@ -239,15 +241,15 @@ var p = new Promise(function () { /* some test code */ });
 
 p.then(function checkAssertions(arg) {
   if (!expected_condition) {
-    $ERROR("failure message");
+    throw new Test262Error("failure message");
```

Comment:
`assert(expected_condition, "assertion message");` is even better than the whole if case.